packages/nouns-contracts/tasks/deploy-and-configure-short-times-dao-v3.ts

```diff
@@ -19,7 +19,7 @@ task(
   .addOptionalParam(
     'auctionReservePrice',
     'The auction reserve price (wei)',
-    1 /* 1 wei */,
+    10000000000000000 /* 0.01 ether */,
```

Comment:
"Never trust a code comment at face value" a wise man once told me

Just a suggestion, had stuff like this bite me in the past, and you would probably want to wrap my snippit in `Number()` to make it an `int` instead of `bigint`

Reply:
you got a good point. updated